src/libopensc/card-incrypto34.c

```diff
@@ -357,6 +357,7 @@ static int incrypto34_create_file(sc_card_t *card, sc_file_t *file)
 				 * generation. */
 			case SC_FILE_EF_LINEAR_VARIABLE_TLV:
 				type[1] = 0xff;
+				/* FALLTHROUGH */
```

Comment:
please avoid the capital letters. according to https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html, `/* fall through */` as used elsewhere in OpenSC should work, too, for `-Wimplicit-fallthrough=3` which gets enabled by `-Wextra` but not for `-Wall`...

Reply:
This sounds reasonably and it works as documented for me. Will fix.